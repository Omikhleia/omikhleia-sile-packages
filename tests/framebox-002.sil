\begin[papersize=a6]{document}
\script[src=packages/framebox]
\script[src=packages/struts]
\nofolios
\neverindent
% Test case: \roughbox basics

A default \roughbox{\strut{}rough box.}

\medskip

A default \roughbox[enlarge=true]{\strut{}rough box} with extra spacing.

\medskip

A filled colored \roughbox[bordercolor=#b94051,fillcolor=220]{\strut{}rough box.}

\medskip

A single-stroked \roughbox[singlestroke=true]{\strut{}rough box.}

\medskip

A vertice-preserving single-stroked \roughbox[singlestroke=true, preserve=true]{\strut{}rough box.}

\medskip

A (strongly-)bowed single-stroked \roughbox[singlestroke=true, bowing=10]{\strut{}rough box.}

\medskip

A (strongly-)rough single-stroked \roughbox[singlestroke=true, roughness=2.5]{\strut{}rough box.}

\medskip

A stretched \roughbox[enlarge=true]{\strut{}rough\hfill box} for test.

\end{document}
