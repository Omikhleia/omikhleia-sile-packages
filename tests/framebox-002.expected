Set paper size 	297.6377985	419.5275636
Begin page
Mx 	14.8819
My 	29.1307
Set font 	Gentium Plus;10;400;;normal;;LTR
T	36 w=5.9521	(A)
Mx 	23.4968
T	71 72 73 68 88 79 87 w=28.9697	(default)
Draw SVG	29.1307	44.7721	10.5957	1.0000	-0.13561 0.11142 m 15.48426 0.57849 31.31116 -0.55849 48.17658 1.79388 c -2.45688 -0.50678 m 9.35468 -0.13195 22.53113 -0.08412 47.58072 0.69496 c 46.85747 0.94469 m 45.33118 4.82022 45.70093 7.06999 46.70457 14.18756 c 46.73387 -0.1147 m 47.06573 6.35102 46.3969 12.50973 47.24079 14.85237 c 47.43269 14.78735 m 29.26456 14.4556 12.15479 16.43669 -3.56439 16.9732 c 47.24867 14.30734 m 27.17125 15.56399 9.95319 15.17209 -2.3218 15.35355 c -3.30201 13.79592 m -2.51143 12.19285 -0.84818 8.35841 -1.81871 1.26375 c -1.39953 14.85207 m -1.91228 11.68675 -1.46977 7.39553 -1.65458 1.12261 c 0 0 0 RG 0.4 w S
Mx 	55.1292
T	85 82 88 74 75 w=24.6826	(rough)
Mx 	82.4745
T	69 82 91 w=15.1367	(box)
Mx 	97.6112
T	17 w=2.2900	(.)
Mx 	14.8819
My 	48.7264
T	36 w=5.9521	(A)
Mx 	23.4876
T	71 72 73 68 88 79 87 w=28.9697	(default)
Draw SVG	48.7264	46.4729	14.5957	1.0000	-0.40142 6.23305 m 14.1724 3.584 27.91226 5.65314 46.88276 2.49991 c 0.42379 4.05291 m 15.12836 4.30887 30.63188 5.2446 45.49944 4.50251 c 45.38092 4.66075 m 47.30299 8.77675 47.22805 11.51073 45.99762 17.95945 c 47.16381 4.43284 m 46.75997 7.44415 46.55555 9.7895 45.96433 18.53066 c 45.31465 19.56474 m 31.28987 19.53509 14.02789 20.50654 0.8316 19.0921 c 46.62887 18.68321 m 30.07381 18.95929 10.8114 18.01608 0.97452 18.17077 c -1.0182 19.51096 m 0.38476 15.75522 1.3333 11.30048 0.15624 3.1038 c -0.16689 19.25955 m 0.24066 15.30619 0.79524 11.79065 0.55804 4.24696 c 0 0 0 RG 0.4 w S
Mx 	57.1108
T	85 82 88 74 75 w=24.6826	(rough)
Mx 	84.4469
T	69 82 91 w=15.1367	(box)
Mx 	104.2372
T	90 76 87 75 w=18.7305	(with)
Mx 	125.6211
T	72 91 87 85 68 w=21.6406	(extra)
Mx 	149.9153
T	86 83 68 70 76 81 74 w=31.2256	(spacing)
Mx 	181.1409
T	17 w=2.2900	(.)
Mx 	14.8819
My 	68.3221
T	36 w=5.9521	(A)
Mx 	23.4939
T	737 79 79 72 71 w=21.0596	(filled)
Mx 	47.2134
T	70 82 79 82 85 72 71 w=30.9180	(colored)
Draw SVG	68.3221	44.7693	10.5957	1.0000	-2.23274 0.70914 m -2.23274 0.70914 -2.23274 0.70914 -2.23274 0.70914 c -2.23274 0.70914 m -2.23274 0.70914 -2.23274 0.70914 -2.23274 0.70914 c -1.72263 2.40489 m -0.90729 1.76643 -0.37738 1.16116 0.51335 -0.02291 c -1.69774 2.62984 m -1.04205 1.68939 -0.36141 0.93921 0.32016 0.14626 c -1.74274 5.40332 m 0.08816 3.77585 1.5468 2.23817 2.05229 0.44113 c -1.92529 5.0853 m -0.46825 3.62542 0.80696 2.18613 2.2437 0.8018 c -0.92929 7.4004 m -0.20266 6.20866 0.64982 4.08883 3.89427 0.87965 c -1.57349 7.9968 m 0.02754 6.16157 0.44312 4.55234 5.09634 0.43669 c -2.21071 10.97249 m 1.12826 6.3008 3.77562 4.14831 6.80846 0.94761 c -1.46013 10.2656 m 1.72444 6.91973 4.33421 2.68613 6.19605 0.19786 c -2.5738 12.83898 m 0.62557 9.84371 6.147 3.94386 6.91516 0.1048 c -2.70586 13.07403 m 0.67495 8.38011 4.03707 5.00213 9.10849 0.71189 c -1.64133 16.14597 m -0.26267 13.0437 2.82108 9.62947 10.22985 0.61742 c -2.55277 15.2216 m 1.01723 10.68135 5.18283 7.08908 11.07039 -0.23917 c 1.35255 12.99256 m 3.32258 10.57028 7.1406 6.80789 13.01166 0.20126 c 0.87028 14.70001 m 3.44444 12.35495 5.62425 8.88185 12.31926 1.04266 c 1.24034 16.79158 m 6.77457 13.48656 8.89312 9.12927 13.72532 0.92791 c 2.07669 14.79361 m 5.1089 10.79177 8.81876 7.57199 14.52005 1.41408 c 5.3684 13.97704 m 9.89407 9.71964 12.57018 4.93259 15.5205 1.96913 c 5.49307 14.12395 m 10.21511 9.10042 13.73879 4.84241 16.98901 -0.43999 c 5.66069 15.41413 m 7.30097 10.70683 12.27888 7.71854 20.57954 1.87621 c 7.33514 15.29298 m 10.59409 10.48205 14.75115 6.76964 18.18068 0.46817 c 8.50239 14.08619 m 12.87553 9.46891 14.85866 7.65641 22.44802 0.43129 c 9.29369 14.9315 m 13.99532 9.19419 18.295 4.22732 20.6531 0.8127 c 12.17832 14.53931 m 15.52671 8.99946 19.54495 5.14047 24.41466 1.00808 c 10.84855 14.82416 m 15.47738 11.12179 18.45123 5.92834 23.22772 1.00404 c 13.80562 14.68069 m 14.93434 11.5284 20.32854 7.90351 26.43822 -0.55889 c 11.86057 15.58158 m 16.8217 11.67981 20.71478 6.9576 26.18104 1.10095 c 16.23116 14.68621 m 20.47669 10.41289 20.98767 4.47259 27.14717 -0.37996 c 15.22247 14.83325 m 18.41451 11.53153 21.42852 8.1769 27.33779 -0.27161 c 15.66581 14.73873 m 19.83836 13.21383 22.47521 6.04822 31.35591 -0.06825 c 18.06709 15.07978 m 21.88701 9.89748 25.38381 4.89409 29.94325 -0.20432 c 18.7789 16.37429 m 25.74149 8.7893 27.00914 5.41107 33.85185 1.84871 c 19.08405 15.92638 m 23.07603 11.66819 25.49237 7.64299 32.53989 0.80471 c 20.61412 15.16531 m 25.65888 9.9886 28.4393 4.19699 35.11093 1.63392 c 21.01296 15.08461 m 24.02563 12.3961 25.99834 9.37883 34.26769 1.15427 c 23.79361 14.31354 m 28.52175 8.50975 32.43221 3.97852 36.13895 0.07145 c 23.12403 15.77616 m 29.32926 9.29716 33.37423 3.70904 36.60777 0.86182 c 23.97103 14.65746 m 29.03773 8.5141 33.08008 5.39012 39.23853 1.13155 c 24.93768 16.34397 m 30.18747 9.6535 34.38406 5.65599 37.84607 0.32347 c 26.68118 15.6445 m 31.76878 10.85795 34.51289 4.30045 42.12593 1.8425 c 27.1639 15.61768 m 30.99346 11.06413 35.60078 5.23419 41.10523 0.02861 c 29.2208 16.12389 m 33.99761 10.2745 36.18733 6.59817 43.33502 -0.10958 c 30.58325 16.17668 m 34.88271 9.70997 39.52217 3.68857 43.31629 -0.7663 c 33.75243 13.81029 m 34.17477 12.5132 39.64707 6.64512 44.3348 1.16086 c 32.9666 15.28788 m 34.85164 10.50143 37.88203 8.04644 43.87017 0.05651 c 33.27155 14.45353 m 35.83065 10.58325 40.98275 4.77597 46.93645 1.37401 c 34.98293 14.38819 m 39.4314 9.17432 42.94877 3.6471 47.17636 1.17951 c 34.17549 16.0755 m 39.82547 9.89962 44.32927 4.48653 46.57947 1.64106 c 35.60431 15.09127 m 39.32481 12.89747 41.96681 8.34616 47.73828 0.98061 c 39.21596 15.799 m 39.77071 11.08402 41.5016 10.56039 49.141 4.81393 c 38.92078 14.77837 m 41.78202 10.80396 44.37385 7.56669 49.00079 2.96838 c 40.07392 14.8599 m 42.54663 13.49084 43.37754 10.04256 49.09027 5.07553 c 40.8157 14.99709 m 42.99596 13.03087 45.75077 10.00193 48.43273 5.95174 c 42.27447 15.32436 m 44.69629 12.75616 46.53905 10.45344 48.85293 8.04012 c 42.67717 14.6698 m 44.39942 12.37727 46.54632 10.44896 48.05365 8.27962 c 44.64903 14.7108 m 45.53658 13.76269 46.36388 12.83424 47.73421 11.57921 c 44.63626 14.98366 m 45.80622 13.45307 47.21737 12.27744 47.96431 11.04076 c 0.86275 G 0.4 w S -2.91824 0.36401 m 17.31083 0.25978 32.69607 1.53789 46.77342 0.64998 c -2.41491 -0.55471 m 12.1774 0.0077 27.69312 0.61046 46.92233 1.33872 c 45.94657 0.82261 m 46.58432 3.52769 47.84795 8.60592 46.41523 13.96119 c 46.1873 0.71171 m 47.14171 4.07802 46.21147 7.88593 47.40363 14.61463 c 48.60115 13.74846 m 29.38269 14.36467 9.35885 15.00106 -1.50305 16.78217 c 47.76397 15.11536 m 33.08199 15.08774 17.7498 15.37009 -1.48572 14.4008 c -1.02741 14.73078 m -2.12897 10.60932 -0.73163 7.12346 -2.41634 -0.39171 c -1.66388 14.51257 m -2.62259 10.8511 -2.64101 5.26546 -2.11508 0.29182 c 0.72549 0.25098 0.31765 RG 0.4 w S
Mx 	80.7912
T	85 82 88 74 75 w=24.6826	(rough)
Mx 	108.1337
T	69 82 91 w=15.1367	(box)
Mx 	123.2705
T	17 w=2.2900	(.)
Mx 	14.8819
My 	86.3221
T	36 w=5.9521	(A)
Mx 	23.4935
T	86 76 81 74 79 72 w=24.2969	(single)
Mx 	47.7904
T	16 w=3.3691	(-)
Mx 	51.1595
T	86 87 85 82 78 72 71 w=30.9912	(stroked)
Draw SVG	86.3221	44.7688	10.5957	1.0000	-3.93882 -0.25541 m 8.24073 -0.7109 19.89239 2.03207 45.91826 -1.05163 c 47.59166 1.00063 m 47.02953 4.87747 47.05325 7.62907 46.13009 14.30526 c 48.1618 16.14658 m 31.05263 13.43989 17.22441 14.4072 -1.35859 13.21637 c -2.3264 15.33963 m -2.89406 11.76274 -1.62125 9.54784 -1.99206 1.0269 c 0 0 0 RG 0.4 w S
Mx 	84.8102
T	85 82 88 74 75 w=24.6826	(rough)
Mx 	112.1522
T	69 82 91 w=15.1367	(box)
Mx 	127.2889
T	17 w=2.2900	(.)
Mx 	14.8819
My 	104.3221
T	36 w=5.9521	(A)
Mx 	23.4848
T	89 72 85 87 76 70 72 w=28.6523	(vertice)
Mx 	52.1371
T	16 w=3.3691	(-)
Mx 	55.5063
T	83 85 72 86 72 85 89 76 81 74 w=44.3555	(preserving)
Mx 	102.5125
T	86 76 81 74 79 72 w=24.2969	(single)
Mx 	126.8094
T	16 w=3.3691	(-)
Mx 	130.1785
T	86 87 85 82 78 72 71 w=30.9912	(stroked)
Draw SVG	104.3221	44.7601	10.5957	1.0000	-2 0.44141 m 18.49825 2.04626 34.39817 0.90062 46.76012 0.44141 c 46.76012 0.44141 m 45.34186 7.18503 46.31153 11.9633 46.76012 15.03711 c 46.76012 15.03711 m 31.91539 14.39248 17.28817 13.93229 -2 15.03711 c -2 15.03711 m -1.8266 9.86097 -1.44284 6.89797 -2 0.44141 c 0 0 0 RG 0.4 w S
Mx 	163.8205
T	85 82 88 74 75 w=24.6826	(rough)
Mx 	191.1538
T	69 82 91 w=15.1367	(box)
Mx 	206.2905
T	17 w=2.2900	(.)
Mx 	14.8819
My 	122.3221
T	36 w=5.9521	(A)
Mx 	23.4854
T	11 w=3.1689	(()
Mx 	26.6543
T	86 87 85 82 81 74 79 92 w=34.3311	(strongly)
Mx 	60.9854
T	16 w=3.3691	(-)
Mx 	64.3545
T	12 w=3.1689	())
Mx 	67.5235
T	69 82 90 72 71 w=26.9873	(bowed)
Mx 	97.1621
T	86 76 81 74 79 72 w=24.2969	(single)
Mx 	121.4590
T	16 w=3.3691	(-)
Mx 	124.8281
T	86 87 85 82 78 72 71 w=30.9912	(stroked)
Draw SVG	122.3221	44.7607	10.5957	1.0000	-0.62377 0.81576 m 6.89197 -1.27516 18.10528 -3.47244 46.39583 -1.47569 c 46.02506 1.58464 m 46.68379 4.50427 48.94399 9.35211 46.90916 16.02465 c 47.73083 13.72397 m 35.12216 12.31089 19.70049 11.15873 -3.05103 14.77415 c -2.91793 14.86037 m -2.20493 11.41179 -1.56166 9.31188 -1.32404 0.4907 c 0 0 0 RG 0.4 w S
Mx 	158.4707
T	85 82 88 74 75 w=24.6826	(rough)
Mx 	185.8046
T	69 82 91 w=15.1367	(box)
Mx 	200.9413
T	17 w=2.2900	(.)
Mx 	14.8819
My 	140.3221
T	36 w=5.9521	(A)
Mx 	23.4856
T	11 w=3.1689	(()
Mx 	26.6546
T	86 87 85 82 81 74 79 92 w=34.3311	(strongly)
Mx 	60.9856
T	16 w=3.3691	(-)
Mx 	64.3548
T	12 w=3.1689	())
Mx 	67.5237
T	85 82 88 74 75 w=24.6826	(rough)
Mx 	94.8579
T	86 76 81 74 79 72 w=24.2969	(single)
Mx 	119.1548
T	16 w=3.3691	(-)
Mx 	122.5239
T	86 87 85 82 78 72 71 w=30.9912	(stroked)
Draw SVG	140.3221	44.7610	10.5957	1.0000	-4.90739 -0.02381 m 14.30952 -2.49497 31.81988 3.56193 47.28154 -2.73975 c 46.63696 3.63738 m 46.53375 1.74821 45.56941 9.37022 47.32859 15.51143 c 50.73462 19.34901 m 28.72495 17.98747 20.38037 19.1399 -2.6618 18.6384 c -1.7995 15.00191 m 1.62136 14.23839 -4.21127 3.79756 -0.91372 -2.38172 c 0 0 0 RG 0.4 w S
Mx 	156.1667
T	85 82 88 74 75 w=24.6826	(rough)
Mx 	183.5009
T	69 82 91 w=15.1367	(box)
Mx 	198.6377
T	17 w=2.2900	(.)
Mx 	14.8819
My 	159.9178
T	36 w=5.9521	(A)
Mx 	23.4766
T	86 87 85 72 87 70 75 72 71 w=39.0332	(stretched)
Draw SVG	159.9178	182.5625	14.5957	1.0000	-1.25114 3.9369 m 63.29801 3.83716 124.66072 3.15394 184.27856 3.30045 c 0.39645 4.97771 m 42.24648 3.60727 84.79778 2.5143 182.08029 4.67653 c 181.61914 4.89739 m 181.80361 9.07093 181.76448 15.11295 183.58019 18.50002 c 183.06841 4.67697 m 183.20724 9.59657 182.31595 13.36028 181.93181 19.20356 c 181.88554 18.13464 m 138.37825 19.00733 92.21393 20.5594 1.78141 20.60976 c 182.84659 18.12532 m 122.37545 17.63252 61.92562 19.35059 0.59041 18.93964 c -0.6696 18.84979 m 1.29077 15.78355 -0.31372 12.12083 0.12311 3.71516 c -0.19898 18.56023 m 0.12901 15.31074 -0.77715 10.84455 -0.31272 4.10522 c 0 0 0 RG 0.4 w S
Mx 	67.1524
T	85 82 88 74 75 w=24.6826	(rough)
Mx 	230.5782
T	69 82 91 w=15.1367	(box)
Mx 	250.3575
T	73 82 85 w=12.0996	(for)
Mx 	265.0997
T	87 72 86 87 w=15.3662	(test)
Mx 	280.4659
T	17 w=2.2900	(.)
End page
Finish
